Sources/Build/LLBuildManifestBuilder.swift

```diff
@@ -626,7 +626,8 @@ extension LLBuildManifestBuilder {
             tempsPath: target.tempsPath,
             objects: try target.objects,
             otherArguments: try target.compileArguments(),
-            sources: target.sources,
+            sourceCount: target.sources.count,
+            fileListPath: try target.writeFileList(target.sources),
```

Comment:
also can we call this sourceListPath or something that make it clear its the sources.

Reply:
Sure, though there is no guarantee that the file list is the sources only.  We can have a file list for libraries as well.